plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergSnapshots.java

```diff
@@ -74,4 +102,13 @@ public void testSelectWithMoreThanOneSnapshotOfTheSameTable()
                 .map(row -> (Long) row.getField(ID_FIELD))
                 .collect(toList());
     }
+
+    private List<String> getQueryIdsFromSnapshotsByCreationOrder(String tableName)
+    {
+        return getQueryRunner().execute(
+                format("SELECT json_extract_scalar(CAST(SUMMARY AS JSON), '$.%s') FROM \"%s$snapshots\"", TRINO_QUERY_ID_NAME, tableName))
+                .getMaterializedRows().stream()
```

Comment:
You could make use of `io.trino.testing.MaterializedResult#getOnlyColumn` for less verbosity.

Reply:
Reverting this back to `getOnlyColumn`. Temporarily there were `null`s in the result stream as CTAS had the `trino_query_id` but not creating an empty table.